modules/2_nbde/files/volume-mount.yaml

```diff
@@ -0,0 +1,26 @@
+- name: playbook for mounting data volume
+  hosts: all
+  become: true
+  tasks:
+    - name: Create directory /data/tangdata if does not exist
+      file:
+        path: /data/tangdata
+        state: directory
+        mode: '0755'
+        recurse: yes
+    - name: create partition
+      parted:
+        device: /dev/mapper/mpathb
+        number: 1
+        flags: [ lvm ]
+        state: present
+        part_end: 4GB
```

Comment:
Make the 4GB a variable that matches the volume_size in var.tfvars

Reply:
used volume_size variable in place of 4GB and used complete 10GB data_volume_size instead of creating partition.